tests/wandb_test.py

```diff
@@ -76,6 +76,25 @@ def test_nice_log_error():
         wandb.log({"no": "init"})
 
 
+def test_log_sanitize(wandb_init_run):
+    wandb.log({"a": 1})
+    assert set(wandb.run.history._data.keys()) == set("a")
+    wandb.log({"a/b": 1})
+    assert set(wandb.run.history._data.keys()) == set("a/b")
+    wandb.log({"a /b": 1})
+    assert set(wandb.run.history._data.keys()) == set("a/b")
+    wandb.log({"a/ b": 1})
+    assert set(wandb.run.history._data.keys()) == set("a/b")
+    wandb.log({"a / b": 1})
+    assert set(wandb.run.history._data.keys()) == set("a/b")
+    wandb.log({"a / b  /   c": 1})
+    assert set(wandb.run.history._data.keys()) == set("a/b/c")
+    wandb.log({"a  / b": {"c  / d / e": {"f  / g ": 1}}})
+    assert set(wandb.run.history._data.keys()) == set("a/b")
+    assert set(wandb.run.history._data["a/b"].keys()) == set("c/d/e")
+    assert set(wandb.run.history._data["c/d/e"].keys()) == set("f/g")
```

Comment:
Hmm, shouldn't this be `wandb.run.history._data["a/b"]["c/d/e"].keys()`?

Reply:
Oh, yes - nice catch.